public/data/textile/materials.json

```diff
@@ -1,235 +1,219 @@
-    "shortName": "Polyester"
+    "recycledProcessId": "994efc14-cc89-526c-b9d0-0d6a99adbd4f"
   },
   {
+    "alias": "ei-pet-r",
     "cff": {
       "manufacturerAllocation": 0.5,
       "recycledQualityRatio": 1.0
     },
     "defaultCountry": "CN",
     "geographicOrigin": "Asie - Pacifique",
-    "id": "ei-pet-r",
-    "materialProcessUuid": "994efc14-cc89-526c-b9d0-0d6a99adbd4f",
+    "id": "8f44c680-1263-4588-b79b-a9e4dc3d8de4",
     "name": "Polyester recyclé",
     "origin": "Synthetic",
     "primary": true,
     "processId": "994efc14-cc89-526c-b9d0-0d6a99adbd4f",
     "recycledFrom": "ei-pet",
-    "recycledProcessUuid": null,
-    "shortName": "Polyester recyclé"
+    "recycledProcessId": null
   },
   {
+    "alias": "ei-pa",
     "cff": null,
     "defaultCountry": "FR",
     "geographicOrigin": "Europe",
-    "id": "ei-pa",
-    "materialProcessUuid": "b2318c3f-6d7b-5d95-b201-6b95470db3b7",
+    "id": "c84280ff-e921-4d5c-92d6-51030bf4f74e",
     "name": "Nylon",
     "origin": "Synthetic",
     "primary": true,
     "processId": "b2318c3f-6d7b-5d95-b201-6b95470db3b7",
     "recycledFrom": null,
-    "recycledProcessUuid": null,
-    "shortName": "Nylon"
+    "recycledProcessId": null
   },
   {
+    "alias": "ei-lin",
     "cff": null,
     "defaultCountry": "FR",
     "geographicOrigin": "Europe",
-    "id": "ei-lin",
-    "materialProcessUuid": "b4e7353b-7963-5d88-b4bc-84bd925f5a86",
+    "id": "a52e486c-b67e-40af-8337-e8143cbf9076",
     "name": "Lin",
     "origin": "NaturalFromVegetal",
     "primary": true,
     "processId": "b4e7353b-7963-5d88-b4bc-84bd925f5a86",
     "recycledFrom": null,
-    "recycledProcessUuid": null,
-    "shortName": "Lin"
+    "recycledProcessId": null
   },
   {
+    "alias": "ei-laine-par-defaut",
     "cff": null,
     "defaultCountry": "CN",
     "geographicOrigin": "Asie - Pacifique",
-    "id": "ei-laine-par-defaut",
-    "materialProcessUuid": "3b45f504-83ac-5d0d-ab96-388850f3ef6f",
+    "id": "1fc3e17d-5661-429d-a150-7986eae16d9d",
     "name": "Laine par défaut",
     "origin": "NaturalFromAnimal",
     "primary": false,
     "processId": "3b45f504-83ac-5d0d-ab96-388850f3ef6f",
     "recycledFrom": null,
-    "recycledProcessUuid": null,
-    "shortName": "Laine par défaut"
+    "recycledProcessId": null
   },
   {
+    "alias": "ei-laine-nouvelle-filiere",
     "cff": null,
     "defaultCountry": "CN",
     "geographicOrigin": "Asie - Pacifique",
-    "id": "ei-laine-nouvelle-filiere",
-    "materialProcessUuid": "40ec17ee-6b44-5ee2-8fbc-559aed52764f",
+    "id": "72e29a61-52a9-4eb5-b3fc-5fb87edea041",
     "name": "Laine nouvelle filière",
     "origin": "NaturalFromAnimal",
     "primary": true,
     "processId": "40ec17ee-6b44-5ee2-8fbc-559aed52764f",
     "recycledFrom": null,
-    "recycledProcessUuid": null,
-    "shortName": "Laine nouvelle filière"
+    "recycledProcessId": null
   },
   {
+    "alias": "ei-coton",
```

Comment:
I don't get why we introduce this semantic alias, especially when the patch aims at avoiding having semantic identifiers; as you say in the description:

> Cette id sémantique peut poser problème si l'on change la matière sous-jacente et que la string ne représente plus le procédé sous jacent.

Shouldn't we only rely on uuids?

Reply:
A refactoring I didn't want to do right now 😆

But even if we do this refactoring, I find there is still an use case for `alias` :
- uuid are obviously needed
- alias are also useful : when we communicate between humans we would intuitively use alias and say 
"Modify the cff manufacturer allocation to 0.7. on `coton-rdp`"  rather than 
"Modify the cff manufacturer allocation to 0.7. on `d3d20384-88fb-4c75-ac12-c647365ac6c5`"

Also as we merge `ingredients` and `materials` in the same `metadata` object, ingredients have an `alias` that is really useful to communicate. 

We use "bellpepper-unheated-greenhouse-fr" instead of "361139ac-1028-4cc8-9c0e-4955c5fa39d3"